packages/cli/src/commands/generate/sdl/templates/sdl.js.template

```diff
@@ -2,7 +2,11 @@ export const schema = gql`
   type ${singularPascalName} {
     ${query}
   }
-
+<% if (enums.length > 0) {%>
+<% enums.forEach((enumDef, idx)=> {%>  enum ${enums[idx].name} {<% enums[idx].values.forEach((enumDefValue, idk)=> { %>
+    ${enums[idx].values[idk].name}<% }) %>
+  }
+<%}) %><% } %>
```

Comment:
@Rosenberg96 - would the following suggestion add the newlines you wanted?

```suggestion
<% if (enums.length > 0) {%>
<% enums.forEach((enumDef, idx)=> {%>  enum ${enums[idx].name} {<% enums[idx].values.forEach((enumDefValue, idk)=> { %>
    ${enums[idx].values[idk].name}<% }) %>
  }

<%}) %><% } %>
```

Reply:
Unfortunately not - this will add a new empty line in all scaffolded `*sd.js` files between the `type` and the `Query` definitions and break those